packages/iiif/src/ns.js

```diff
@@ -1,15 +1,15 @@
-import namespace from '@rdfjs/namespace'
+import rdf from '@zazuko/env'
 
 const ns = {
-  iiifImage: namespace('http://iiif.io/api/image/3#'),
-  iiifPrezi: namespace('http://iiif.io/api/presentation/3#'),
-  dctypes: namespace('http://purl.org/dc/dcmitype/'),
-  exif: namespace('http://www.w3.org/2003/12/exif/ns#'),
-  dcterms: namespace('http://purl.org/dc/terms/'),
-  schema: namespace('https://schema.org/'),
-  as: namespace('http://www.w3.org/ns/activitystreams#'),
-  oa: namespace('http://www.w3.org/ns/oa#'),
-  rdf: namespace('http://www.w3.org/1999/02/22-rdf-syntax-ns#'),
+  iiifImage: rdf.namespace('http://iiif.io/api/image/3#'),
+  iiifPrezi: rdf.namespace('http://iiif.io/api/presentation/3#'),
+  dctypes: rdf.ns.dctypes,
```

Comment:
From what I see, the `rdf.ns.dctypes` does not exist.

By looking at the vocabulary repo, I found this: https://github.com/zazuko/rdf-vocabularies/blob/236121b082f9d7d833130ce3ae0afb1ea7176d06/ontologies/dcmitype/meta.nt#L6

So this should do the trick:

```suggestion
  dctypes: rdf.ns.dcmitype,
```

Reply:
Ah, it did cross my mind to add type checking while I was on this package. Would have caught that